src/select.ts

```diff
@@ -0,0 +1,30 @@
+import 'rxjs/add/operator/map';
```

Comment:
rather than using the add syntax here (which has sideeffects that will propagate down to applications), how about using the operator directly? - see https://github.com/angular/angular/blob/master/modules/angular2/src/facade/async.ts#L58